docs/ref/models/indexes.txt

```diff
@@ -0,0 +1,90 @@
+=====================
+Model index reference
+=====================
+
+.. module:: django.db.models.indexes
+   :synopsis: Built-in index types.
+
+.. currentmodule:: django.db.models
+
+Index classes define a specific database index which can be created through
+them. This document explains the API references of :class:`BaseIndex` which is
+the base class for all other index classes. This includes the `index options`_
+and `built-in indexes`_ Django offers. For now the indexes can be created in
+the database via the :class:`~django.db.migrations.operations.CreateIndex` class as follows::
+
+    from django.db import migrations, models
+
+    class Migration(migrations.Migration):
+
+        operations = [
+            migrations.CreateIndex(
+                model_name='Author',
+                models.Index('name', 'age', name='my_index_name') # An index on the name and age fields
```

Comment:
I'd also replace `'name'` with `'title'` to prevent confusion that `'name'` is a column and not the name of the index.